### ISSUE TITLE
Upgrading to Symfony 3.3 fails

### ISSUE THREAD

**Issue by reyostallenberg** [Sylius/Sylius#8149]
When upgrading to Symfony 3.3 the composer update gives the following runtime exception:
```
  [RuntimeException]  
  An error occurred when executing the "'cache:clear --no-warmup'" command:                                                                    
  PHP Fatal error:  Uncaught Symfony\Component\DependencyInjection\Exception\InvalidArgumentException: Tag "<argument>" with type="service" has no or empty "id" attribute in "vendor/sylius/sylius/src/Sylius/Bundle/ThemeBundle/DependencyInjection/../Resources/config/services/integrations/assets.xml". in vendor/symfony/symfony/src/Symfony/Component/DependencyInjection/Loader/XmlFileLoader.php:484      
```

**Comment by venyii** [comment]
This particular error was fixed in #8031

**Comment by reyostallenberg** [comment]
Okay, I'm very sorry for the duplication.

Thanks a lot
